docker/README.md

```diff
@@ -0,0 +1,8 @@
+
+The open eye license file is not included and must be passed in as a mount point.
+I have an `oe_license.txt` file in `/home/mmh/.openeye/`.
+The container expects `oe_license.txt` in `/perses/` because we set `ENV OE_LICENSE=/perses/oe_license.txt` in the Dockerfile.
+
+To do that use the `--mount` option.
+For example and to to test:
+docker run --rm --mount type=bind,source=/home/mmh/.openeye/,target=/perses/,readonly mmh42/perses:0.9.1  python -c "import openeye; assert openeye.oechem.OEChemIsLicensed(), 'OpenEye license checks failed!'"
```

Comment:
Can you change `/home/mmh` to `$HOME`?

Reply:
Done! I used `~` originally and docker complained, but `$HOME` worked!